code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm

```diff
@@ -52,3 +52,47 @@
 	holder.pixel_x = 24
 	holder.pixel_y = 24
 	hud_list[XENO_EVASION_HUD] = holder
+
+/mob/living/carbon/xenomorph/runner/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
+	. = ..()
+	if(!ishuman(over))
+		return
+	if(!src.back)
+		balloon_alert(over,"This runner isn't wearing a saddle!")
+		return
+	if(!do_after(over, 3 SECONDS, NONE, src))
+		return
+	var/obj/item/storage/backpack/marine/duffelbag/xenosaddle/saddle = src.back
+	src.dropItemToGround(saddle,TRUE)
```

Comment:
dont use src.var, it's implicit. just use var

Reply:
done

Reply:
done